packages/backend/src/models/whiteboard.ts

```diff
@@ -437,6 +437,20 @@ export class Whiteboard {
         });
         break;
       }
+      case OperationType.DENY_PENDING_CLIENT: {
+        const { deniedClient } = op.data;
+        const clientWasPending = this.pendingClients.delete(deniedClient.id);
+        if (!clientWasPending) {
+          throw new Error(
+            `Client ${deniedClient.id} was not pending for whiteboard ${this.id}`
+          );
+        }
+        deniedClient.send({
```

Comment:
I think here the client.fsm should also be reverted to NO_WHITEBOARD. Becaue currently the frontend goes back to showing whiteboard selection form, but the backend does not allow this operation becaue the client is stuck in pending.

Reply:
Great catch, will fix!